suites/pacific/iscsi/tier-1_rbd_test-iscsi_multiple_deployment.yaml

```diff
@@ -0,0 +1,208 @@
+#===============================================================================================
+#--------------------------------------
+# Conf: conf/pacific/cephadm/sanity-cephadm.yaml
+# Verifies : https://bugzilla.redhat.com/show_bug.cgi?id=2007516
+#
+# 1. Deploy 5.x cluster with mgr,mon, osd services
+# 2. Create pool and deploy ISCSI with 4 gateways
+# 3. Check "Ceph orch ls" for service status
+# 4. Perform Removal of service and deploy ISCSI for 2-3 times
+# 5. Check the cluster health and Ceph orch ls
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap with custom dashboard credentials
+      desc: Bootstrap with initial-dashboard user and password option
+      module: test_bootstrap.py
+      polarion-id: CEPH-83573719
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          registry-url: registry.redhat.io
+          custom_image: true
+          mon-ip: "node1"
+          orphan-initial-daemons: true
+          skip-monitoring-stack: true
+          initial-dashboard-user: admin123
+          initial-dashboard-password: admin123
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
```

Comment:
These test cases are basically used for deployment of desired cluster. Can't it be done in single test case using config as - https://github.com/red-hat-storage/cephci/pull/1679/files#diff-48264722f3668b92a38798438ac704dfc6dcdc73e0cee33cb65ff276a80ae0baR18

Reply:
Ack. Noted. The same has been followed.